### ISSUE TITLE
Missing coordinate system

### ISSUE THREAD

**Issue by noahdasanaike** [dickoa/rgeoboundaries#18]
Under the newest version of the package, geoboundaries("Japan", adm_lvl = "adm0") returns an object with a missing CRS.

**Comment by DanRunfola** [comment]
@noahdasanaike Thank you for reporting this error - we think we found the issue (we weren't saving the CRS correctly in the geoJSON files being built; this only appears to have impacted geoJSONS at this time, but we're investigating further). 

If we're correct about where the error was occuring, this should be fixed later this evening after we rebuild the dataset.  I'll reply back to this issue as soon as we have a confirmation.

(We were able to replicate this error in R!).

-Dan

**Comment by DanRunfola** [comment]
(Oh, and FYI this is because the new API uses the updated geoBoundaries data for the first time - the old API was stuck on geoBoundaries 3.0.0!  So you should see some large data improvements).

In the short term, if you need to project all CRS's are EPSG:4326

**Comment by noahdasanaike** [comment]
Thanks so much, @DanRunfola!

**Comment by DanRunfola** [comment]
Just to provide an update on this: we did find the error, but our initial solution didn't work (we're not quite sure why - possibly something with the binary we use to build the geoJSON, mapshaper.  We're digging a little deeper into that.).  I just did a number of tests and found a temporary solution using geopandas, which is now implemented and will be running as soon as our local cluster has room to do the processing.  I'd now estimate the ETA for a fix for this is ~tomorrow early morning Eastern time.

Thanks for the report and patience!

Dan
